app/executor.php

```diff
@@ -276,7 +276,7 @@ function getStorageDevice($root): Device
                 throw new Exception('Failed to create build container', 500);
             }
 
-            $orchestration->networkConnect($runtimeId, App::getEnv('OPEN_RUNTIMES_NETWORK', 'appwrite_runtimes'));
+            $orchestration->networkConnect($runtimeId, App::getEnv('OPEN_RUNTIMES_NETWORK', 'appwrite-runtimes'));
```

Comment:
Can you test this locally as well ? This was working perfectly fine before ( even in the current Appwrite 0.15 ) and I'm sceptical about changing this.

Reply:
I can test, but even if it worked, it's wrong and network name is not `appwrite_runtimes` but `appwrite-runtimes`. It might have worked without assigning the proper network. but we need to make sure we assign the proper network

Reply:
@christyjacob4 figured out the problem, there was a bug with Docker compose 2.8.0 which was what gitpod was using at the time, which has been reverted and fixed in docker-compose 2.9.0 so I will revert back this change.
https://github.com/docker/compose/releases/tag/v2.8.0

Reply:
awesome 👍